### PR TITLE
[WIP] Add InversifyJS

### PR DESCRIPTION
**Pull request by BrendanAnnable**

### CODE REVIEW

File: src/client/components/localisation/view.tsx [hunk 1]
```diff
@@ -3,19 +3,22 @@ import { runInAction } from 'mobx'
 import { observer } from 'mobx-react'
 import * as React from 'react'
 import { WebGLRenderer } from 'three'
+import { lazyInject } from '../../../inversify.config'
```
Comment (MonicaOlejniczak) [inline src/client/components/localisation/view.tsx]: Add a webpack alias for this. Can it also have a better name while you're at it? I'm not fond of it being called a config.

File: src/client/components/localisation/view.tsx [hunk 2]
```diff
@@ -3,19 +3,22 @@ import { runInAction } from 'mobx'
...
 
 interface LocalisationViewProps {
-  model: LocalisationModel,
-  presenter: LocalisationPresenter
+  model: LocalisationModel
 }
 
 @observer
 export class LocalisationView extends React.Component<LocalisationViewProps, any> {
+  @lazyInject(LocalisationPresenter)
```
Comment (MonicaOlejniczak) [inline src/client/components/localisation/view.tsx]: Are we always going to lazy inject? I feel like we shouldn't know the way it's being injected in the localisation view.

File: src/client/components/localisation/view.tsx (hunk 1 above)
Comment (BrendanAnnable) [inline src/client/components/localisation/view.tsx]: Coming back to this in another PR! Name suggestions welcome, but `inversify.config` is the standard/recommended.

File: src/client/components/localisation/view.tsx (hunk 2 above)
Comment (BrendanAnnable) [inline src/client/components/localisation/view.tsx]: Fixed

**Pull request merged**